cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -1229,40 +1237,82 @@ def test_api_v2_export_dataset(self):
                     continue
                 dump_format_name = dump_format.DISPLAY_NAME
                 with self.subTest(format=dump_format_name):
-                    project = self._create_project(projects['main'])
-                    pid = project['id']
-                    images = self._generate_task_images(3)
-                    tasks['task in project #1']['project_id'] = pid
-                    self._create_task(tasks['task in project #1'], images)
-                    images = self._generate_task_images(3, 3)
-                    tasks['task in project #2']['project_id'] = pid
-                    self._create_task(tasks['task in project #2'], images)
+                    if dump_format_name in [
+                        'Cityscapes 1.0', 'LFW 1.0', 'Market-1501 1.0',
+                        'MOT 1.1', 'TFRecord 1.0'
+                    ]:
+                        self.skipTest("TO-DO: fix bug for this formats")
```

Comment:
`skipTest` and `subTest` do not play well together. I did some testing, and when `pytest` is used as the test runner, `skipTest` just skips the entire rest of the test (when `unittest` is used as the runner, only the subtest is skipped, but the whole test is _marked_ skipped in the log, which is not great either).